rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -0,0 +1,245 @@
+        elif KEY_OR in step.keys():
+            self._parse_or_statement(step)
+        elif KEY_SLOT_NAME in step.keys():
+            self._parse_slot(step)
+        elif KEY_ACTION in step.keys():
+            self._parse_action(step)
+        elif KEY_CHECKPOINT in step.keys():
+            self._parse_checkpoint(step)
+        elif KEY_METADATA in step.keys():
+            pass
+        else:
+            raise_warning(
+                f"Issue found in {self.source_name}: \n"
+                f"Found an unexpected step in the story description:\n"
+                f"{step}\n"
+                "It will be skipped.",
+                docs=DOCS_URL_STORIES,
+            )
+
+    def _parse_plain_story_user_utterance(self, step: Dict[Text, Any]) -> None:
+
+        if self.use_e2e:
+            # TODO
+            pass
+        else:
+            utterance = self._parse_raw_user_utterance(step)
+            self.current_step_builder.add_user_messages([utterance])
+
+    def _parse_or_statement(self, step):
+        utterances = []
+
+        for sub_step in step.get(KEY_OR):
+            if KEY_PLAIN_STORY_USER_UTTERANCE in sub_step.keys():
+                utterance = self._parse_raw_user_utterance(sub_step)
+                if utterance:
+                    utterances.append(utterance)
+            else:
+                raise_warning(
+                    f"Issue found in {self.source_name}: \n"
+                    f"`OR` statement can only have `{KEY_PLAIN_STORY_USER_UTTERANCE}` "
+                    f"as a sub-element. This step will be skipped:\n"
+                    f"`{sub_step}`\n",
+                    docs=DOCS_URL_STORIES,
+                )
+                return
+
+        self.current_step_builder.add_user_messages(utterances)
+
+    def _parse_raw_user_utterance(self, step: Dict[Text, Any]) -> UserUttered:
+
+        user_utterance = step.get(KEY_PLAIN_STORY_USER_UTTERANCE, "").strip()
+
+        if not user_utterance:
+            raise_warning(
+                f"Issue found in {self.source_name}: \n"
+                f"User utterance cannot be empty."
+                "This story step will be skipped:\n"
+                f"{step}",
+                docs=DOCS_URL_STORIES,
+            )
+
+        raw_entities = step.get(KEY_PLAIN_STORY_ENTITIES, [])
+
+        final_entities = []
+        for entity in raw_entities:
+            if isinstance(entity, dict):
+                for key, value in entity.items():
+                    final_entities.append({"entity": key, "value": value})
+            else:
+                final_entities.append({"entity": entity, "value": ""})
+
+        if not user_utterance.startswith(INTENT_MESSAGE_PREFIX):
+            raise_warning(
+                f"Issue found in {self.source_name}: \n"
+                f'User intent "{user_utterance}" should start with '
+                f'"{INTENT_MESSAGE_PREFIX}"',
+                docs=DOCS_URL_STORIES,
```

Comment:
we should also tell them that we skip it then, shouldn't we?

Reply:
I think yes, missed that.

Reply:
we still should tell them that we skipped